06-classification_continued.Rmd

```diff
@@ -404,8 +496,8 @@ knn_results <- workflow() %>%
 knn_results
```

Comment:
The results here show both `accuracy` and `roc_auc`. We should tell them which rows to pay attention to, what `roc_auc` is, and that it is beyond the scope of this book but they might learn about it in more advanced classed. There are some other columns we should probably explain too - or at least tell them to ignore, such as `.estimator`, `n`, `std_error` and `.config`. I think in general, anytime we show them a data frame summary from an analysis, we need to explain it fully, and point out relevant and non-relevant information.

Reply:
done